pkgs/applications/misc/qwalkingpad/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, lib
+, qtbase
+, qtconnectivity
+, wrapQtAppsHook
+, fetchFromGitHub
+, cmake
+}:
+
+stdenv.mkDerivation rec {
+  pname = "qwalkingpad";
```

Comment:
The author named it `QWalkingPad` with camel case. But I guess it's better to match the name of the binary, right?